Makefile

```diff
@@ -1,32 +1,33 @@
 TOPDIR=$(dir $(lastword $(MAKEFILE_LIST)))
+TASKID_FILE := $(shell mktemp)
 
 SUBDIRS=\
 	standard-controller \
 	address-space-controller \
-	agent \
-	mqtt-gateway \
-	mqtt-lwt \
-	topic-forwarder \
-	auth-plugin \
-	api-server \
-	service-broker \
-	broker-plugin \
-	none-auth-service \
-	controller-manager \
-	console-init \
-	console-httpd \
-	iot-auth-service \
-	iot-device-registry-file \
-	iot-device-registry-datagrid \
-	iot-http-adapter \
-	iot-lorawan-adapter \
-	iot-mqtt-adapter \
-	iot-proxy-configurator \
-	iot-sigfox-adapter \
-	iot-tenant-service \
-
-
-all: $(SUBDIRS)
+	#agent \
```

Comment:
These modules shouldn't be commented out

Reply:
ah.. sorry that was my short cut testing!  I'll fix.